approve-all.ps1

```diff
@@ -0,0 +1,17 @@
+param (
```

Comment:
I believe approve-all.ps1 it should go to Build folder. I know it's not there (not visible), but I do have some automation in place (just need to improve it for showing on GitHub), as it was just a temporary solution, that somehow stayed. 

![image](https://user-images.githubusercontent.com/15063294/205486443-f2c21ec5-a2d1-4339-96a7-f5a4a5d4449a.png)